multisig-transactions-monitor/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "forta-compound-finance-multisig-transaction-agent",
+  "name": "forta-compound-finance-multisig-transaction-bot",
```

Comment:
This string is used to determine the Bot ID by the `forta-agent` package.  Changing any characters in the string will result in a different Bot ID, so when the Bot is deployed, it will not update the old one but rather deploy a new one.  Unfortunately, in this case it is unavoidable because the word `agent` appears in the string.

What this means is that for this Bot, as well as any other Bots that updates made to the `name` field, we will need to manually disable the old Bot.

Reply:
just added